docs/paying-for-lit/overview.md

```diff
@@ -0,0 +1,60 @@
+# Paying for the Usage of the Lit Network
+
+Like other decentralized networks, Lit has a certain amount of computation available for users that's metered to allow for a responsive network with nodes that are able to stay in-sync with one another.
+
+In order to use the paid pre-production ([Datil-test](../connecting-to-a-lit-network/testnets#datil-test)) and production ([Datil](../connecting-to-a-lit-network/mainnets#datil)) Lit networks, you must reserve capacity on the network. This is done using by minting Capacity Credits, and by making use of the Lit Relayer and/or the Payment Delegation Database.
```

Comment:
Might be helpful to call out exactly which kinds of activities on Lit require payment. Connecting to the network? Generating a Session Sig? Reading chain state? Minting a PKP? Creating a Wrapped Key? Signing? Decrypting? Encrypting? Running a Lit Action?

Not sure if this needs to be a table, or just a call out. But if it's not a table, then we'll need to mention whether an activity requires payment in their respective sections.